TYPO3.Neos/Classes/TYPO3/Neos/TypoScript/PluginImplementation.php

```diff
@@ -149,6 +149,12 @@ public function evaluate() {
 			$pluginResponse = new Response($parentResponse);
 
 			$this->dispatcher->dispatch($this->buildPluginRequest(), $pluginResponse);
+			
+      /* Put header from sub-request into main-request */
```

Comment:
With the current Flow CGL the comment should look like:

    // Put header from ...
    if (...) {
    }

Inline comments should generally be avoided and could be replaced by extracting the code into a method with a speaking name.